### PR TITLE
updating aind-data-schema pin

### LINKED ISSUE #26: update aind-data-schema pin

### PR DESCRIPTION
**Pull request by dyf**
resolves #26

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -17,8 +17,9 @@ readme = "README.md"
 dynamic = ["version"]
 
 dependencies = [
-    'aind-data-schema==0.31.8',
-    'pydantic<2.7'
+    'aind-data-schema==0.38.0',
+    'pydantic<2.7',
+    'tzlocal'
```
Comment (jtyoung84) [inline pyproject.toml:22]: If this is just being used in the tests, we should add it to the optional dependencies.

Comment (dyf) [reply pyproject.toml]: cool

**Review by alejoe91 — APPROVED** [review]
Thansk @dyf 

Tested on the ephys pipeline and everything is running smoothly

**Comment by alejoe91** [comment]
@dyf went ahead and merged this since I'm doing a round of pipeline updates including metadata ;)

**Pull request merged**